actgen/change_q.py

```diff
@@ -53,6 +53,8 @@ def parse_args(self):
                             help='Path to the result directory to save model files')
         parser.add_argument('--tag', type=str, default='default_exp',
                             help='A tag for the current experiment, used as a subdirectory name for saving models')
+        parser.add_argument('--gpu', default=False, action='store_true',
```

Comment:
Let's change this arg to `--disable-gpu`, and have the default behavior automatically detect whether there's a gpu and use it if it's available, _unless_ `--disable-gpu` has been specified.